packages/remark-mdx/test/fixtures/block-parsing.md

```diff
@@ -41,8 +41,8 @@ Weeeeee!
 
 # Hello, world!
 
-<Foo>{
-  const num = 1+1
-
-  return <h2>{num}</h2>
-}</Foo>
+<Foo>
+  {
+    <h2>{1+1}</h2>
+  }
+</Foo>
```

Comment:
This was never one expression: JSX only allows expressions, not whole programs.
And I couldn’t get `<Tag>{` to play nice with micromark, hence the line between.